content/en/platform/corda/1.5/cenm/jira-setup.md

```diff
@@ -10,48 +10,44 @@ title: JIRA Set-Up
 ---
 
 
-# JIRA Set-Up
+# Jira Setup
 
-The following are the instructions on setting up the JIRA in order to satisfy minimal requirements from the point of view
-of the Identity Manager Certificate Signing Request (CSR)/Certificate Revocation Request (CRR) approval workflow. This will enable
-a user to integrate such JIRA deployment with the CENM CSR/CRR approval workflow plugin.
+To integrate a Jira deployment with the CENM CSR/CRR approval workflow plugin, it must meet the requirements of the Identity Manager Certificate Signing Request (CSR)/Certificate Revocation Request (CRR) approval workflow.
 
+## Configure projects and users
 
-## User Set-Up
+1. Create two `Process Management` projects: one for CSRs and one for CRRs. The final settings for both projects should look like this:
 
-Create a user with an administrative permissions and make sure it is assigned to the default groups:
-`jira-administrators` and `jira-software-users`. The following shows an example for a `Test` user setup.
+![jira 6](/en/images/jira-6.png "jira 6")
 
-![jira 7](/en/images/jira-7.png "jira 7")
+2. Create a user with permission to:
+    * Create tickets
+    * Change ticket statuses
 
-## Project Set-Up
 
-Create two projects (one for Certificate Signing Requests and one for Certificate Revocation Requests) of the type of
-`Process Management`. The following shows final settings for both projects.
+3. Assign the user to the CSR and CRR projects.
 
-![jira 6](/en/images/jira-6.png "jira 6")
+4. Supply the user credentials to the [CENM identity manager configuration](identity-manager.html#jira-workflow).
 
 ## Project Workflow
```

Comment:
```suggestion
## Configure projects' workflow
```
It's a task, right? Not sure if you're configuring or just checking the projects' configuration though.